src/frontend/src/pages/HomePage/UpcomingDeadlines.tsx

```diff
@@ -100,7 +100,7 @@ const UpcomingDeadlines: React.FC = () => {
         </FormControl>
       }
     >
-      <Container fluid>{workPackages.isLoading ? <LoadingIndicator /> : fullDisplay}</Container>
+      <Grid container>{workPackages.isLoading ? <LoadingIndicator /> : fullDisplay}</Grid>
```

Comment:
![image](https://user-images.githubusercontent.com/29521172/196088568-2a100f14-b4b0-40ec-82ec-5df0789c0728.png)

The version with the grid doesn't have the box we have on prod; we probably wanna keep some kind of separators between the work packages